core/src/main/java/com/linecorp/armeria/server/RouteBuilder.java

```diff
@@ -461,7 +466,9 @@ public String toString() {
 
     private static PathMapping getPathMapping(String pathPattern) {
         requireNonNull(pathPattern, "pathPattern");
-
+        if (Strings.isNullOrEmpty(pathPattern)) {
```

Comment:
`pathPattern` is never `null`. Use `pathPattern.isEmpty()`.

Reply:
Fixed 0879163 😄